### PR TITLE
Add Secrets test project to sln

### PR DESCRIPTION
**Pull request by pakrym**
So it runs as part of nightly builds

### CODE REVIEW

File: sdk/keyvault/Microsoft.Azure.KeyVault.sln [hunk 1]
```diff
@@ -18,12 +18,15 @@ EndProject
...
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.KeyVault.TestFramework", "Microsoft.Azure.KeyVault.TestFramework\src\Microsoft.Azure.KeyVault.TestFramework.csproj", "{6F5823CD-D1D7-4DF4-BE61-F07FD8134433}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.KeyVault.TestFramework", "Microsoft.Azure.KeyVault.TestFramework\src\Microsoft.Azure.KeyVault.TestFramework.csproj", "{424F477A-5470-4FC9-B74F-233B671F6A77}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.KeyVault.WebKey", "Microsoft.Azure.KeyVault.WebKey\src\Microsoft.Azure.KeyVault.WebKey.csproj", "{643372AD-2DBC-4395-95B9-61183D3F2904}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.KeyVault.WebKey.Tests", "Microsoft.Azure.KeyVault.WebKey\tests\Microsoft.Azure.KeyVault.WebKey.Tests.csproj", "{6F5823CD-D1D7-4DF4-BE61-F07FD813440A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Security.KeyVault.Secrets.Tests", "Azure.Security.KeyVault.Secrets\tests\Azure.Security.KeyVault.Secrets.Tests.csproj", "{FE4C57FD-0D75-485D-9FBE-622DDBCEF30C}"
```
Comment (maririos) [inline sdk/keyvault/Microsoft.Azure.KeyVault.sln]: Is there another way to do this? Mixing track 1 projects with track 2 seems odd

Comment (maririos) [reply sdk/keyvault/Microsoft.Azure.KeyVault.sln]: Could we add a step in the .yml so that it picks or looks for the Azure.Security.KeyVault.sln ?

Comment (pakrym) [reply sdk/keyvault/Microsoft.Azure.KeyVault.sln]: Oh, I didn't notice that we have two sln files. I'll redo the PR>

Comment (pakrym) [reply sdk/keyvault/Microsoft.Azure.KeyVault.sln]: Hey @weshaggard, what should we do here? We have separate solutions for track 1/2 in keyvault and eventhubs, how do we make sure both are built and tested?

Comment (weshaggard) [reply sdk/keyvault/Microsoft.Azure.KeyVault.sln]: We have plans to use a traversal project and not the solution to build and run as we just chatted we haven't done that yet for the tests, we will get that fixed.

For the keyvault live tests we also need to setup the necessary secrets to run them that has never been done.

**Pull request closed without merging**